controllers/bootstrap/init.go

```diff
@@ -185,11 +185,33 @@ func (b *Bootstrap) CrossplaneCloudOperator(instance *apiv3.CommonService) error
 		if err := b.installCrossplaneOperator(); err != nil {
 			return err
 		}
+	} else {
+		if err := b.DeleteCrossplaneCloudSubscription(b.CSData.MasterNs); err != nil {
+			return err
+		}
 	}
 
 	return nil
 }
 
+// DeleteCrossplaneCloudSubscription deleted crossplane & cloud operator subscription when bedrockshim set to false or CS CR is removed
+func (b *Bootstrap) DeleteCrossplaneCloudSubscription(namespace string) error {
+	klog.Infof("Trying to delete ibm-crossplane-operator in %s", namespace)
+	if err := b.deleteSubscription("ibm-crossplane-operator-app", namespace); err != nil {
```

Comment:
I think before we delete the crossplane-operator, we should remove its CRs as well.

Reply:
`Configuration` & `Lock`? they are created after the subscription: https://github.com/IBM/ibm-common-service-operator/blame/f550579f88dd93ee3e5ac3b13661222c1d3c7f3a/controllers/bootstrap/init.go#L621
do we need to remove all its CRs?

Reply:
@horis233 could you mind confirming it? Should we uninstall `Configuration` & `Lock`?

I thought we should remove all the stuff installed by ourselves during the uninstallation.

And yes, because they are created after subscription(after the operator is installed), we should delete them before removing the operator.

Reply:
Yes, We should delete them before removing the operator, Otherwise crossplane operands will still be there.